website/src/content/docs/ja/guides/integrate-in-editor.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: Biome とエディタの統合
+description: エディタやIDEで Biome を統合する方法を学びます
+---
+
+## 公式プラグイン
+
+これらは、Biome チームによってメンテナンスされているプラグインで、 [Biome organization](https://github.com/biomejs) によって管理されています。
+
+### VS Code
+
+Biome のプラグインを使うと、以下のことができます:
+
+* ファイルの保存時やコマンドを実行したときに、フォーマットを実行する
+* ファイルの静的解析をし、コードの修正を適用する
+
+
+Visual Studio Code マーケットプレイスページで、[Biome VS Code extension](https://marketplace.visualstudio.com/items?itemName=biomejs.biome) をインストールしてください。
+
+Biome をデフォルトのフォーマッターにするには、[supported file](/ja/internals/language-support/) を開いて、以下の手順に従ってください:
+
+- **コマンドパレット**を開く（<kbd>Ctrl</kbd>/<kbd title="Cmd">⌘</kbd>+<kbd title="Shift">⇧</kbd>+<kbd>P</kbd> または 表示 → コマンドパレット）
+- **ドキュメントのフォーマット...** を選択
+- **デフォルトのフォーマッターの設定...** を選択
+- Biomeを選択
+
+### IntelliJ
+
+Biome IntelliJ 向けのプラグインをインストールするには、[プラグインの公式ページ](https://plugins.jetbrains.com/plugin/22761-biome) にアクセスするか、以下の手順に従ってください:
+
+**JetBrains IDEs の場合:**
+
+1. IntelliJ IDEA を開く。
+2. **Settings/Preferences** に移動する。
+3. 左のメニューから **Plugins** を選択する。
+4. **Marketplace** タブをクリックする。
+5. 検索ボックスに "Biome" と入力し、**Install** をクリックする。
+6. IDEを再起動して、プラグインを有効にする。
+
+**ディスクからの場合:**
+
+1. releases タブからプラグインの .zip をダウンロードする。
+2. `⌘Сmd,` を押して IDE の設定を開き、**Plugins** を選択する。
+3. プラグインページで、**Settings** ボタンをクリックし、**Install Plugin from Disk...** をクリックする。
+
+## サードパーティのプラグイン
+
+これらは、各種エディタで使用できる、コミュニティによってメンテナンスされているプラグインです:
+- [`neovim`](https://neovim.io/): [`nvim-lspconfig`](https://github.com/neovim/nvim-lspconfig/) をインストールして、[instructions](https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#biome) に従ってください
+- [`helix`](https://helix-editor.com/): [マニュアル](https://github.com/biomejs/biome/blob/main/editors/helix/manual.md) の手順に従ってください
+- [`coc-biome`](https://github.com/fannheyward/coc-biome): [`coc.nvim`](https://github.com/neoclide/coc.nvim) 向けの Biome 拡張機能
+
+:::note
+ここにのっていないエディタのプラグインがありますか？ぜひ PR を送って、ここに追加してください！
+:::
+
+## 自分でプラグインを書く
+
+Biome は [LSP](https://microsoft.github.io/language-server-protocol/) を第一級でサポートしています。
+エディタが LSP を実装している場合、Biome の統合をシームレスに行うことができます。
+
+### LSP プロキシを使う
+
+Biome には `lsp-proxy` というコマンドがあります。 実行すると、Biome は2つのプロセスを生成します:
+- 要求された操作を実行する [デーモン](/ja/internals/architecture#デーモン-wip)
```

Comment:
I'm not sure what url should be linked here.

In original document, the link is `https://biomejs.dev/internals/architecture/#daemon`.
But `#daemon` doesn't exist on that page.

I changed the link's tag to currently exists one.

Reply:
I seem the current link is no problem 👍